radio/src/gui/480x272/menu_model.cpp

```diff
@@ -118,7 +118,7 @@ void editName(coord_t x, coord_t y, char * name, uint8_t size, event_t event, ui
             killEvents(event);
             break;
           }
-          // no break
+          [[fallthrough]];
```

Comment:
This is legal cpp?

Reply:
Yes this is what I learnt last week
Sonar doesn't understand `// no break` but it understands `[[fallthrough];`

Reply:
yes. That is C++17. But similar things exist in C+11 already: http://en.cppreference.com/w/cpp/language/attributes

Reply:
I think this attribute is C++14